src/etc/inc/util.inc

```diff
@@ -45,9 +45,11 @@ function isvalidpid($pidfile) {
 
 function is_process_running($process) {
 	$output = "";
-	exec("/bin/pgrep -anx " . escapeshellarg($process), $output, $retval);
-
-	return (intval($retval) == 0);
+	if ($process != "") {
```

Comment:
This should probably be `!empty()` instead.

Reply:
fixed

Reply:
hm, closed after `git push`..

new: https://github.com/pfsense/pfsense/pull/4318